easybuild/easyconfigs/b/BBMap/BBMap-38.87-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,41 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Pau Ruiz <pau.ruiz@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '38.87'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['22ab642b8af88faf208a56763158da895004c5231df572d3163ce52fbfb63240']
+
+dependencies = [('Java', '11', '', True)]
```

Comment:
While this works, the usage of `True` for indicating that this dependency is at the system toolchain level is now deprecated. Please replace with `dependencies = [('Java', '11', '', SYSTEM)]`.

Reply:
done! please check @casparvl